ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -863,6 +863,56 @@ router bgp {{ router_bgp.as }}
+{%         endif %}
+{%         if router_bgp.address_family_link_state.bgp.missing_policy.direction_out_action is arista.avd.defined %}
+      bgp missing-policy direction out action {{ router_bgp.address_family_link_state.bgp.missing_policy.direction_out_action }}
+{%         endif %}
+{%         for peer_group in router_bgp.address_family_link_state.peer_groups | arista.avd.natural_sort('name') %}
+{%             if peer_group.activate is arista.avd.defined(true) %}
+      neighbor {{ peer_group.name }} activate
+{%             elif peer_group.activate is arista.avd.defined(false) %}
+      no neighbor {{ peer_group.name }} activate
+{%             endif %}
+{%             if peer_group.missing_policy.direction_in_action is arista.avd.defined %}
+      neighbor {{ peer_group.name }} missing-policy direction in action {{ peer_group.missing_policy.direction_in_action }}
+{%             endif %}
+{%             if peer_group.missing_policy.direction_out_action is arista.avd.defined %}
+      neighbor {{ peer_group.name }} missing-policy direction out action {{ peer_group.missing_policy.direction_out_action }}
+{%             endif %}
+{%         endfor %}
+{%         for neighbor in router_bgp.address_family_link_state.neighbors | arista.avd.natural_sort('ip_address') %}
+{%             if neighbor.activate is arista.avd.defined(true) %}
+      neighbor {{ neighbor.ip_address }} activate
+{%             endif %}
+{%             if neighbor.missing_policy.direction_in_action is arista.avd.defined %}
+      neighbor {{ neighbor.ip_address }} missing-policy direction in action {{ neighbor.missing_policy.direction_in_action }}
+{%             endif %}
+{%             if neighbor.missing_policy.direction_out_action is arista.avd.defined %}
+      neighbor {{ neighbor.ip_address }} missing-policy direction out action {{ neighbor.missing_policy.direction_out_action }}
+{%             endif %}
+{%         endfor %}
+{%         if router_bgp.address_family_link_state.path_selection is arista.avd.defined %}
+{%             if router_bgp.address_family_link_state.path_selection.roles.producer is arista.avd.defined(true) %}
+      path-selection
+{%             endif %}
+{%             if router_bgp.address_family_link_state.path_selection.roles.consumer is arista.avd.defined or router_bgp.address_family_link_state.path_selection.roles.propagator is arista.avd.defined %}
+{%                 set path_selection_roles = "path-selection role" %}
+{%                 if router_bgp.address_family_link_state.path_selection.roles.consumer is arista.avd.defined(true) %}
+{%                     set path_selection_roles = path_selection_roles ~ " consumer" %}
+{%                 endif %}
+{%                 if router_bgp.address_family_link_state.path_selection.roles.propagator is arista.avd.defined(true) %}
+{%                     set path_selection_roles = path_selection_roles ~ " propagator" %}
+{%                 endif %}
```

Comment:
This might lead to `path-selection role` CLI (with no role set), if both are set to `False`

Reply:
no because of the surrounding if

Reply:
The surrounding `if` (line 904) is not checking for `true` so if I put `false` in any of them, this section would run, but fail both the checks underneath, leading to invalid cli.

Reply:
added `true` in [c15b6fc](https://github.com/aristanetworks/ansible-avd/pull/3211/commits/c15b6fc3d0da3a11bedc0b807f41fe19bcc97bdb)